ledger-tool/src/bigtable.rs

```diff
@@ -308,30 +308,39 @@ async fn confirm(signature: &Signature, verbose: bool) -> Result<(), Box<dyn std
 
 pub async fn transaction_history(
```

Comment:
The changes in here are actually not directly related :)

Big table chokes if you ask it to return hundreds of thousands of row keys at once, so here a single BigTable query is just being broken up into many smaller queries